docs/onboarding/azure-devops-pipelines.md

```diff
@@ -819,3 +820,141 @@ In order to configure audit stream for Azure Monitor, identify the following inf
     3. In the Run Pipelines dialog window, enter the first 4 digits of your new subscription configuration file name (4 is usually enough of the GUID to uniquely identify the subscription) between the square brackets in the `subscriptions` parameter field. For example: `[802e]`.
 
     4. In the Run Pipelines dialog window, click the `Run` button to start the pipeline.
+
+## Migration
+
+### Populate management group hierarchy from your environment
+
+You can migrate to the management group hierarchy implemented in v0.9.0 by populating the hierarchy from your existing Azure environment.  By migrating to the hierarchy, you can take advantage of simplified configuration without modifying Bicep templates.  To generate the hierarchy:
+
+1. Login to Azure CLI.
+2. Execute the following command
+
+    ```bash
+    # Replace this value with your environment.  This id is the same as Azure Active Directory ID.
+    TENANT_ROOT_GROUP_ID='<< ENTER YOUR TENANT ROOT GROUP ID >>'
+    
+    az account management-group show --name $TENANT_ROOT_GROUP_ID -e -r | jq 'recurse(.children[]?) |= del(select(.type == "/subscriptions")) | del(..|nulls) | recurse(.children[]?) |= {"id":.name, "name":.displayName, "children": (.children // [] )}'
+    ```
+
+    This command will:
+
+    * Use Azure CLI to retrieve the management group structure recursively from `Tenant Root Group`.  Response from Azure CLI will include subscriptions.
+    * Use `jq` to
+      * Remove all objects that are of type: `/subscription`
+      * Removing the subscription objects leaves `null` in place in arrays so we then delete the nulls references
+      * Recurse through the intermediate json and select `name`, `id` and `children` (if children is null, then mark it as an empty array `[]`).
+
+    Output:
+
+    ```json
+    {
+      "id": "<< ENTER YOUR TENANT ROOT GROUP ID >>",
```

Comment:
Remove the word `ENTER` from the sample output, to just show `<<YOUR TENANT ROOT GROUP ID>>`